mopidy/core/actor.py

```diff
@@ -136,6 +140,103 @@ def tags_changed(self, tags):
             self.playback._stream_title = title
             CoreListener.send('stream_title_changed', title=title)
 
+    def setup(self):
```

Comment:
Should we also have an if-already-called check in here, and then just log a warning and return?